js/join.js

```diff
@@ -243,6 +243,36 @@ function previewFiller() {
   previewHeardAbout.innerHTML = window.localStorage.getItem('foundFrom');
 }
 
+function sendJoinData() {
+  let firstName = window.localStorage.getItem('firstName');
+  let lastName = window.localStorage.getItem('lastName');
+  let city = window.localStorage.getItem('city');
+  let state = window.localStorage.getItem('state');
+  let country = window.localStorage.getItem('country');
+  let introduction = window.localStorage.getItem('introduction');
+  let skills = window.localStorage.getItem('skills');
+  let college = window.localStorage.getItem('college');
+  let forFun = window.localStorage.getItem('forFun');
+  let whyRds = window.localStorage.getItem('whyRds');
+  let foundFrom = window.localStorage.getItem('foundFrom');
+  let funFact = window.localStorage.getItem('funFact');
+  let data = {
+    firstName,
+    lastName,
+    city,
+    state,
+    country,
+    introduction,
+    skills,
+    college,
+    forFun,
+    whyRds,
+    foundFrom,
+    funFact,
+  };
+  return data;
+}
```

Comment:
Could we generate it from a constants array please?
`['firstName', 'lastName', 'city', 'state'...]`